avatar.css

```diff
@@ -0,0 +1,124 @@
+body{
+    position: relative;
+}
+.container{
+    background-color:skyblue ;
+    padding: 3px;
+    display: flex;
+    gap: 5px;
```

Comment:
You may give it in rems maybe

Reply:
but rem would be to large as 1rem equals to 16px